### DISCUSSION TITLE
Error registering and creating new client instance

### DISCUSSION THREAD

**Discussion by sun2dayo** [eldy/SellYourSaas#193] [General]
Hello
First of all I want to thank you for the interesting project.
I'm trying to test the project implementation, using 2 servers, one master and another Deployement.
But I'm encountering error when creating first client instance.
When confirming the registration to create the client instance (in myaccont.mydomain.com) and I get the following error:
"...action_deploy_undeploy.sh for action deployall on demo with1.mydomain.com return 12, so remote agent returns http code 562.."
However the customer account is created and Deployment status Processing.
I checked all the Master server side configurations as well as the Deployment one. Everything looks like the tutorial. I analyzed the conditions of action_deploy_undeploy.sh, remote_server_launcher.sh , register_instance.php and I can't find the faults in the settings.
Do you have any idea of the cause of this error? Thanks in advance.
![Registration-error-](https://user-images.githubusercontent.com/95107558/155687680-7876fad7-5601-454e-a002-9e0b0f8a2e98.png)

**Comment by hregis** [comment]
@sun2dayo and what do you have in /var/log/remote_server.log ?

**Reply by sun2dayo** [reply]
Hi @hregis 
Thanks for your feedback.
I checked /var/log/remote_server.log and other settings.
I realized that the problem was associated with the Mount disk.
There was a mistake on my part related to the option of the disk dedicated or not to the 
backup.
So I managed to solve this problem.
However, I find another error right after that, which prevents me from creating the client instance, which I think is associated with NS records.
Error: "Connection refused (141.XX.XX.XX@suporte.with1.mydomain.com/ddrdSw24dfo9)".
![deployerror](https://user-images.githubusercontent.com/95107558/155847040-6e70d4ff-33a1-4064-a35e-546f7e2c7302.png)

About the  /var/log/remote_server.log:
...
2022-02-26 10:55:32 Process of action deployall of suporte.with1.mydomain.com for user osunip8gjftg finished with no error
...

Thanks

**Comment by vladismernc** [comment]
Hello, I have the same problem but in my case the remote_server.log says "Failed to find archive directory /home/jail/archives-test," and the directory exists but does not have any files inside. How could I fix it?
